tritium-processor/src/main/java/com/palantir/tritium/processor/Methods.java

```diff
@@ -66,9 +67,9 @@ static IdentityHashMap<MethodElements, String> methodStaticFieldName(
         IdentityHashMap<MethodElements, String> result = new IdentityHashMap<>();
         for (MethodElements method : instrumentedMethods) {
             String methodName = method.element().getSimpleName().toString();
-            String originalMethodFieldName = CaseFormat.LOWER_CAMEL
+            String originalMethodFieldName = Strings.nullToEmpty(CaseFormat.LOWER_CAMEL
                     .converterTo(CaseFormat.UPPER_UNDERSCORE)
-                    .convert(methodName);
+                    .convert(methodName));
```

Comment:
`methodName` shouldn't ever be null, so I'm not sure exactly why this flags as possibly null. Going to switch this to a `checkNotNull` to enforce that

Reply:
I believe the intent is that convert input is nullable, and the outputs nullability matches the input, but the annotations don't have a clear way to express that

Reply:
Yeah, I was just a little surprised compiler couldn't verify this was always non-null as it flowed through. There's some interesting bits in [`Converter`](ParametricNullness) and [`com.google.common.base.NullnessCasts#uncheckedCastNullableTToT`](https://github.com/google/guava/blob/master/guava/src/com/google/common/base/NullnessCasts.java#L55)